tests/system/providers/google/cloud/dataproc_metastore/example_dataproc_metastore.py

```diff
@@ -153,52 +159,6 @@
     )
     # [END how_to_cloud_dataproc_metastore_export_metadata_operator]
 
-    # [START how_to_cloud_dataproc_metastore_create_backup_operator]
-    backup_service = DataprocMetastoreCreateBackupOperator(
-        task_id="create_backup",
-        project_id=PROJECT_ID,
-        region=REGION,
-        service_id=SERVICE_ID,
-        backup=BACKUP,
-        backup_id=BACKUP_ID,
-        timeout=TIMEOUT,
-    )
-    # [END how_to_cloud_dataproc_metastore_create_backup_operator]
-
-    # [START how_to_cloud_dataproc_metastore_list_backups_operator]
-    list_backups = DataprocMetastoreListBackupsOperator(
-        task_id="list_backups",
-        project_id=PROJECT_ID,
-        region=REGION,
-        service_id=SERVICE_ID,
-    )
-    # [END how_to_cloud_dataproc_metastore_list_backups_operator]
-
-    # [START how_to_cloud_dataproc_metastore_delete_backup_operator]
-    delete_backup = DataprocMetastoreDeleteBackupOperator(
-        task_id="delete_backup",
-        project_id=PROJECT_ID,
-        region=REGION,
-        service_id=SERVICE_ID,
-        backup_id=BACKUP_ID,
-        timeout=TIMEOUT,
-    )
-    # [END how_to_cloud_dataproc_metastore_delete_backup_operator]
-
-    # [START how_to_cloud_dataproc_metastore_restore_service_operator]
-    restore_service = DataprocMetastoreRestoreServiceOperator(
-        task_id="restore_metastore",
-        region=REGION,
-        project_id=PROJECT_ID,
-        service_id=SERVICE_ID,
-        backup_id=BACKUP_ID,
-        backup_region=REGION,
-        backup_project_id=PROJECT_ID,
-        backup_service_id=SERVICE_ID,
-        timeout=TIMEOUT,
-    )
-    # [END how_to_cloud_dataproc_metastore_restore_service_operator]
-
     # [START how_to_cloud_dataproc_metastore_delete_service_operator]
     delete_service = DataprocMetastoreDeleteServiceOperator(
```

Comment:
Should you not add a `trigger_rule=TriggerRule.ALL_DONE` to this operator since it is cleaning up a resource?

Reply:
It is done outside START/END documentation markers (to have cleaner example) in the line 172: delete_service.trigger_rule = TriggerRule.ALL_DONE

Reply:
Ah yes of course, we do this too! I think the new line spacing which groups it with the delete_bucket assignment made me miss it.